examples/Extensions/AddSitelinks.php

```diff
@@ -0,0 +1,339 @@
+<?php
```

Comment:
Sometimes you use "sitelink" and "site link". Based on the Google Help terminology, I think it should be "sitelink" everywhere.

Please consider double checking the camel case too if you make changes

Reply:
Thanks for catching this. I myself told Devin to use sitelink consistently too.